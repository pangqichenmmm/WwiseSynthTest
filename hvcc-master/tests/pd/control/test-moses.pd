#N canvas 430 62 241 214 10;
#X obj 32 18 loadbang;
#X obj 32 115 moses -1;
#X obj 32 147 print LHS;
#X obj 97 147 print RHS;
#X msg 72 81 -230;
#X obj 32 79 f 0;
#X obj 32 40 t b b;
#X connect 0 0 6 0;
#X connect 1 0 2 0;
#X connect 1 1 3 0;
#X connect 4 0 1 0;
#X connect 5 0 1 0;
#X connect 6 0 5 0;
#X connect 6 1 4 0;
