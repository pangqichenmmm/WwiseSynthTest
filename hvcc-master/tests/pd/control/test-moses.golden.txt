[@ 0.000] LHS: -230
[@ 0.000] RHS: 0
